tests/libtest/env.py

```diff
@@ -8,78 +8,77 @@
 import json
 import tempfile
 import shutil
+import socket
 import sys
 
-from . import get_free_port
 from thrift_client_to_db import get_viewer_client
-from thrift_client_to_db import get_server_client
 from thrift_client_to_db import get_auth_client
 
 from functional import PKG_ROOT
 from functional import REPO_ROOT
 
 
+def get_free_port():
```

Comment:
It is absolutely not a good way, due to that problem.
It is also absolutely a method that has been moved from another location to this place.

But since we only ever set up a single server (excluding some certain more instristic test cases), this method is used way less times than before this patch. AFAIK, we call this 4 times, as opposed to the around 20-25 (postgresql AND server port was retrieved by this!) previously.